### PR TITLE
Translations / Folder for German     de

### PR DESCRIPTION
**Pull request by Black-Z18XE**
Basically all the folders / files for the    de   translations. Also translated 2 files already, the rest is just the english files.

Username Black-Z18XE   
Discord: Addy#6585   
Language(ISO):  de

### CODE REVIEW

**Review by leonardssh — APPROVED** [review]
LGTM!

**Comment by leonardssh** [comment]
Hit me up when the PR is ready to merge. 👍

**Comment by Black-Z18XE** [comment]
I guess that will take ages @leonardssh. Maybe its smarter to merge it every now and then, when some parts are completed. For example once im done with the callbacks

idk really

**Comment by leonardssh** [comment]
@Black-Z18XE take your time. I'll merge this when everything its done. 👍

**Comment by Southclaws** [comment]
Remove the English files from the PR and we can merge it.

Duplicating 600+ pages is not idea, and will make tracking progress very difficult.

**Comment by Black-Z18XE** [comment]
@Southclaws can you tell me how this works, im pretty new to github. Or should i add you to the repository, so you can help me with it ?

**Comment by Black-Z18XE** [comment]
Or do you mean, i should remove the english from the de folder? I was going to change them over time

**Comment by Black-Z18XE** [comment]
Im a little confused now, i think at first the structure should be taken from the english and then all files should be translated over time.

**Comment by Southclaws** [comment]
Yes the structure must be the exact same as English, but don't literally upload English documents into the German directory for the aforementioned reasons.

You can keep that locally or on a separate branch but in pull requests, it's basically impossible to review or track progress when all 600+ files are in a single pull request and 99% of them are still English content.

**Comment by Black-Z18XE** [comment]
Just found some time to remove the english files for you. I guess its now ready for the first merge.
@Southclaws

File: docs/translations/de/client/ClientCommands.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+Titel: "Client Befehle"
```
Comment (Southclaws) [inline docs/translations/de/client/ClientCommands.md:2]: please don't convert the metadata tags, the system looks for `title` and `description` it doesn't translate struct field names as these are static and defined in code.

Comment (Black-Z18XE) [reply docs/translations/de/client/ClientCommands.md:2]: Ah i see its changed already.
From now on, how do i go on about adding to the translations ? 
Do i simply add the files to this PR or how does it work?

File: docs/translations/de/server/addfileshere [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (Southclaws) [inline docs/translations/de/server/addfileshere:1]: what is this file?

**Comment by Southclaws** [comment]
Should be good to merge, just need to delete that random empty file

**Comment by Southclaws** [comment]
Thank you for the contribution!

**Pull request merged**